src/org/thoughtcrime/securesms/ConversationListItem.java

```diff
@@ -173,7 +173,7 @@ public void bind(@NonNull ThreadRecord thread,
     }
 
     fromView.setCompoundDrawablesWithIntrinsicBounds(
-        Prefs.isChatMuted(getContext(), (int) chatId)? R.drawable.ic_volume_off_grey600_18dp : 0,
+        Prefs.isChatMuted(dcContext.getChat(chatId))? R.drawable.ic_volume_off_grey600_18dp : 0,
```

Comment:
this database roundtrip looks unnecessarily expensive to me, esp. as it is executed for every line in the chatlist. we have the chat-record as legacy thread-object, we can copy the state there without using the database again.

but nm, i have already a fix, did that during testing/review :)

Reply:
did a thing: https://github.com/deltachat/deltachat-android/pull/1356/commits/8b19b81c5b4078e2ad54612eeb1de3d860987b06